src/dataMap/metaManager/metaSchema.js

```diff
@@ -330,6 +341,7 @@ export default () => {
      * @memberof Options#
      * @type {number|number[]|string|string[]|Array<undefined>|Function}
      * @default undefined
+     * @category Core, ManualRowResize, AutoRowSize
```

Comment:
As we assumed on slack, it should be associated only with Core.

Reply:
done